<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_PermissionWidget</class>
 <widget class="QWidget" name="DB_PermissionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_13">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDatabaseName">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leHostName">
       <property name="placeholderText">
        <string>Local database</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>User/Group</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget_3">
          <layout class="QGridLayout" name="gridLayout_7">
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Users</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <property name="spacing">
               <number>3</number>
              </property>
              <item row="0" column="0">
               <widget class="RB_ToolButtonBar" name="tbbUser" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="RB_TableView" name="tvUser"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_9">
             <property name="horizontalSpacing">
              <number>9</number>
             </property>
             <property name="verticalSpacing">
              <number>3</number>
             </property>
             <item row="0" column="2">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>User Name</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Telephone</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="leFirstName"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="leLastName"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>First Name</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Email</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Last Name</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="leTelephone"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="leEmail"/>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>End</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QDateEdit" name="deEndUser">
               <property name="displayFormat">
                <string>yyyy-MM-dd</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Start</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QDateEdit" name="deStartUser">
               <property name="displayFormat">
                <string>yyyy-MM-dd</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Password</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="leUserName"/>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lePassword"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_4">
          <layout class="QGridLayout" name="gridLayout_10">
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>3</number>
           </property>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>System Group</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="RB_IdxLineEdit" name="ileUserSystemGroup"/>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>User Groups</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_11">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <property name="spacing">
               <number>3</number>
              </property>
              <item row="0" column="0">
               <widget class="RB_ToolButtonBar" name="tbbUserGroup" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="RB_TableView" name="tvUserGroup"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Project/Group/Permission</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_14">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget_1">
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Permission Project List</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>1</number>
              </property>
              <property name="topMargin">
               <number>1</number>
              </property>
              <property name="rightMargin">
               <number>1</number>
              </property>
              <property name="bottomMargin">
               <number>1</number>
              </property>
              <property name="spacing">
               <number>1</number>
              </property>
              <item row="0" column="0">
               <widget class="RB_ToolButtonBar" name="tbbProject" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="RB_TableView" name="tvProject"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="horizontalSpacing">
              <number>9</number>
             </property>
             <property name="verticalSpacing">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Project Number</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="leProjectNumber"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Status</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QComboBox" name="cbStatus">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Description</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="leDescription"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Start</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QDateEdit" name="deStartProject">
               <property name="displayFormat">
                <string>yyyy-MM-dd</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>End</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QDateEdit" name="deEndProject">
               <property name="displayFormat">
                <string>yyyy-MM-dd</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="leCustomer"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Customer</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="lblPerspectiveProject">
               <property name="text">
                <string>Perspective Project</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1" colspan="3">
              <widget class="RB_IdxLineEdit" name="ilePerspectiveProject"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Permission Groups</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <property name="spacing">
               <number>3</number>
              </property>
              <item row="0" column="0">
               <widget class="RB_ToolButtonBar" name="tbbGroup" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="RB_TableView" name="tvGroup"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>System Group</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="RB_IdxLineEdit" name="ileProjectSystemGroup"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RB_ToolButtonBar</class>
   <extends>QWidget</extends>
   <header>rb_toolbuttonbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RB_TableView</class>
   <extends>QTableView</extends>
   <header>rb_tableview.h</header>
  </customwidget>
  <customwidget>
   <class>RB_IdxLineEdit</class>
   <extends>QLineEdit</extends>
   <header>rb_idxlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tvUser</tabstop>
  <tabstop>leFirstName</tabstop>
  <tabstop>leLastName</tabstop>
  <tabstop>leTelephone</tabstop>
  <tabstop>leEmail</tabstop>
  <tabstop>leUserName</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>deStartUser</tabstop>
  <tabstop>deEndUser</tabstop>
  <tabstop>tvUserGroup</tabstop>
  <tabstop>ileUserSystemGroup</tabstop>
  <tabstop>leDatabaseName</tabstop>
  <tabstop>leHostName</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tvProject</tabstop>
  <tabstop>leProjectNumber</tabstop>
  <tabstop>leDescription</tabstop>
  <tabstop>leCustomer</tabstop>
  <tabstop>ilePerspectiveProject</tabstop>
  <tabstop>cbStatus</tabstop>
  <tabstop>deStartProject</tabstop>
  <tabstop>deEndProject</tabstop>
  <tabstop>tvGroup</tabstop>
  <tabstop>ileProjectSystemGroup</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
